src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -79,12 +82,25 @@
 </kd-help-section>
 
 <kd-help-section>
-  <md-switch ng-model="ctrl.isExternal" class="md-primary"
+  <md-checkbox ng-model="ctrl.isExternal" class="md-primary no-ink"
```

Comment:
I added no-ink in order to get the blue color (when checked) consistent with the rest of the page. Angular's default checkbox is pink: https://material.angularjs.org/latest/demo/checkbox